src/main/java/org/phoebus/olog/ElasticConfig.java

```diff
@@ -58,6 +58,12 @@ public class ElasticConfig {
     @Value("${elasticsearch.sequence.index:olog_sequence}")
     private String ES_SEQ_INDEX;
 
+    @Value("${archive.modified.entries:true}")
```

Comment:
Could this be opt-in instead of opt-out by default? 

Also tech-wise: This value is repeated in two different classes. It could be extracted so a configuration bean.

Reply:
We could look into disabling the REST end point for editing log entries, then we can make it so that the feature is opt in. However, we are using update operations for some of the reply and group operations.
Generally, I wanted the archiving of logs on any modify to be the default behavior since that is the recommended practice.

For the `@ConfigurationProperties`, our application.properties files consists of properties for various hierarchy... related to spring, elastic, auth, olog specific I found the configuration POJO looked messy... but I can revisit this again.